recipes/oslo.context/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "oslo.context" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ee05a37829ec797e371a4a25cfebbce42c2ec1bb63ed40028761ff7b83958627
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pbr !=2.1.0,>=2.0.0
+    - pip
+    - python >=3.6
+  run:
+    - python !=2.1.0,>=3.6
```

Comment:
This constraint was supposed to be on pbr :-)

Reply:
Good catch! Fixed...